includes/platform-checkout/class-platform-checkout-utilities.php

```diff
@@ -122,10 +123,16 @@ public function get_persisted_available_countries() {
 	 * @return array
 	 */
 	public function get_woopay_available_countries() {
-		$last_check = get_option( self::AVAILABLE_COUNTRIES_LAST_CHECK_KEY );
+		$last_check_option = get_option( self::AVAILABLE_COUNTRIES_LAST_CHECK_KEY );
+		$timezone          = new \DateTimeZone( wp_timezone_string() );
+		$current_date      = new \DateTime( 'now', $timezone );
 
-		if ( $last_check && gmdate( 'Y-m-d' ) === $last_check ) {
-			return $this->get_persisted_available_countries();
+		if ( false !== $last_check_option ) {
```

Comment:
Not a blocker, it's unlikely to happen but if `$last_check_option` returns `null` then this will break.  If we want to check, maybe we can check if `$last_check_option` is a date format.